asio/include/asio/fiber.hpp

```diff
@@ -0,0 +1,1553 @@
+// Copyright (c) 2021 Klemens D. Morgenstern
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+#ifndef ASIO_FIBER_HPP
+#define ASIO_FIBER_HPP
+
+#if defined(_MSC_VER) && (_MSC_VER >= 1200)
+# pragma once
+#endif // defined(_MSC_VER) && (_MSC_VER >= 1200)
+
+#include "asio/detail/config.hpp"
+
+#include <boost/context/fiber.hpp>
+#include <optional>
```

Comment:
Does this require C++17?

Reply:
It does atm, I guess it should work with C++11 (which is what boost.context requires)